src/main/java/frc/robot/subsystems/FalconDriveSubsystem.java

```diff
@@ -153,8 +154,8 @@ public void drive(final double moveRequest, final double turnRequest) {
       leftFront.set(ControlMode.Velocity, leftETPerS);
       rightFront.set(ControlMode.Velocity, rightETPerS);
     } else {
```

Comment:
You should work on getting PID working on the drive.
That's the way to prevent wheelies since you can set the max acceleration as part of the Smart Motion.